fmriprep/reports/core.py

```diff
@@ -34,7 +34,6 @@ def _load_config(self, config):
         from yaml import safe_load as load
 
         settings = load(config.read_text())
```

Comment:
I think we can get rid of this subclass and use `nireports.assembler.report.Report` directly.